packages/sync/src/config/DataSyncConfig.ts

```diff
@@ -35,6 +36,20 @@ export interface DataSyncConfig {
    */
   storage?: PersistentStore<PersistedData>;
 
+  /**
+   * [Modifier]
+   *
+   * The storage the offline client will use to persist offline mutations
+   */
+  offlineStorage?: PersistentStore<OfflineItem>;
+
+  /**
+   * [Modifier]
+   *
+   * The storage options you want the default driver to use. Ignored if `storage` is passed in config.
+   */
+  storageOptions?: StorageOptions;
```

Comment:
Feels really strange to have localforage specific options (undocumented here) together with the ability to swap storage. 

No strong opinion but from my point of view it is important to figure out defaults and let people pass custom storage if they want it. 

There will be much better storage solutions for Cordova and React native so localforage will not always be the answer, but it could be sensible default for web and cordova for now.

Reply:
OK, yeah good point. I will set it to configure defaults unless they provide the storage themselves.